packages/ag-charts-website/src/content/docs/animation/_examples/initial-load/main.ts

```diff
@@ -177,11 +177,11 @@ const pieOptions: AgChartOptions = {
     ],
 };
 
-let chart = AgEnterpriseCharts.create(barOptions);
+const options = barOptions; // Fix for non-vanilla transformers
```

Comment:
Nit: Does this column end up in the examples? It might be confusing for users if so?

Reply:
Removed the comment and renamed `barOptions` to just `options`.